CONTRIBUTING.md

```diff
@@ -7,11 +7,11 @@ Your contributions are welcome!
 One of the main ways you can contribute to this repository is by signing (and starring!) the repo to show your support. Follow these steps to sign:
 
 1. Click on the edit icon of the README.md
-1. Add your name and affiliation as you would like it to appear
-1. Create a pull request:
+2. Add your name and affiliation as you would like it to appear
```

Comment:
```
1. hello
1. world
```
will render as 
1. hello
1. world

Reply:
Yes, it works. I think use ordered list item will be more better.